app/build.gradle

```diff
@@ -30,4 +32,6 @@ dependencies {
     compile 'com.michaelpardo:activeandroid:3.1.0-SNAPSHOT'
     compile 'com.jakewharton:butterknife:6.1.0'
     compile 'org.fluttercode.datafactory:datafactory:0.8'
+    compile 'org.parceler:parceler-api:0.2.16-SNAPSHOT'
+    provided 'org.parceler:parceler:0.2.16-SNAPSHOT'
```

Comment:
read about parceler here: https://github.com/johncarl81/parceler.
Parcelerer is a Java interface that allows serialization of objects. It's one of the ways to pass objects around activities (the alternatives include having a shitty global param to hold the objects). 

Parceler allows you to add annotations that automatically implement the interface.